custom.py

```diff
@@ -56,18 +58,22 @@ def load_custom(self, dataset_dir, subset):
         dataset_dir: Root directory of the dataset.
         subset: Subset to load: train or val
         """
-        # Add classes. We have only one class to add.
-        self.add_class("object", 1, "Car")
-        self.add_class("object", 2, "Truck")
+        # Add classes. We have only 3 classes to add.
+        self.add_class("object", 1, "beer")
+        self.add_class("object", 2, "foam")
+        self.add_class("object", 3, "foam_beer")
 
         # Train or validation dataset?
         assert subset in ["train", "val"]
         dataset_dir = os.path.join(dataset_dir, subset)
 
         # We mostly care about the x and y coordinates of each region
-        annotations1 = json.load(open('D:\\env_with_tensorflow1.14\\all_maskrcnn\\maskrcnn_truck_car\\Dataset\\train\\train_json.json'))
-        # print(annotations1)
-        annotations = list(annotations1.values())  # don't need the dict keys
+        annotations = json.load(open(os.path.join(dataset_dir,
+                                 'via_region_data.json')))
```

Comment:
Is dit de naam van je json file met annotaties?

Reply:
ja, in elke folder staat nu een json file met als naam 'via_region_data'